sympy/polys/polyroots.py

```diff
@@ -1013,9 +1013,8 @@ def _try_heuristics(f):
                                 for root in _try_decompose(f):
                                     _update_dict(result, root, 1)
                         else:
-                            for factor, k in factors:
-                                for r in _try_heuristics(Poly(factor, f.gen, field=True)):
-                                    _update_dict(result, r, k)
+                            for r in _try_heuristics(Poly(factors[0][0], f.gen, field=True)):
```

Comment:
Now `f` has only one factor `factors[0][0]`, and they have the same roots. Could `f` itself be used instead of `Poly(factors[0][0], f.gen, field=True)`?